src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.test.jsx

```diff
@@ -29,8 +31,23 @@ describe('AudioPlayer blocks OnDemandHeading', () => {
     expect(spanWithAriaRoleText).toContainElement(brandTitle);
   });
 
+  it('should not emit an h1 when ariaHidden is true', () => {
```

Comment:
What's this for please?

Reply:
We only want one H1 per page - based on the Zeplin designs, the H1 should be the visually hidden heading at the top of the page.

Here, we're testing that if the visible headline is told to be aria-hidden, it doesn't create an H1